external/source/rdll_template/README.md

```diff
@@ -0,0 +1,27 @@
+# Reflective DLL Project Template
+This is a project template for Microsoft's Visual Studio to aid in the creation
+of Reflective DLLs.
+
+## Installation
+To install the template, copy the template file to the Visual Studio templates
+directory. For a default Visual Studio 2019 installation the command would be:
+
+```
+copy "Reflective DLL.zip" "%USERPROFILE%\Documents\Visual Studio 2019\Templates\ProjectTemplates"
```

Comment:
Do you not need to unzip this first?

Reply:
No, you do not need to unzip it. The zip file *is* the template, so I can clarify that.

Reply:
Confirmed the correction makes this more clear, thanks @zeroSteiner!